build.gradle.kts

```diff
@@ -186,16 +186,18 @@ allprojects {
     publishing {
         publications {
             create<MavenPublication>("Maven") {
+                // Copy data from the kotlin component, notably dependencies
+                from(components.getByName("kotlin"))
+
                 artifactId = project.name
                 version = projectVersion
 
+                // Publish Sources Jar
                 artifact(tasks.remapSourcesJar.get().archiveFile) {
                     builtBy(tasks.remapSourcesJar)
                     this.classifier = "sources"
                 }
```

Comment:
Are sources not part of the kotlin component ?

Reply:
no, thats usually done by loom (i.e `java { withSourcesJar() }`)

since the dokka artifact already needs to be attached, it makes more sense imo to to just leave it here as part of the whole publication